requirements.txt

```diff
@@ -16,7 +16,7 @@ Werkzeug==2.2.2  # It's crucial to specify this package's version to make flask
 flask==2.2.3
 browser-cookie3
 rouge_score==0.1.2
-litellm==1.41.13  # Unifies interface from most LLM APIs
+litellm # Unifies interface from most LLM APIs
```

Comment:
I think it's better to have a version specified, so that we can control when the update is made. Otherwise, silent changes could happen. I could be fine with it, but at least I'd vote for adding a version before merging to main.